package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hemera-dynamodb-store",
-  "version": "1.0.0",
+  "version": "0.0.2",
```

Comment:
You also released it? When not reset 0.0.0

Reply:
Yes I have published it on npm. What version should I now write to package.json ? 0.0.0 ?

Reply:
no, when you already published it then we will drive with semver.